provider/lxd/environ.go

```diff
@@ -165,6 +165,11 @@ func (env *environ) Destroy() error {
 			return errors.Trace(err)
 		}
 	}
+	if env.storageSupported() {
+		if err := destroyModelFilesystems(env); err != nil {
+			return errors.Annotate(err, "destroying LXD volumes for model")
```

Comment:
"LXD filesystems" maybe?
The underlying artefact may be an LXD volume but we're presenting to the user a Juju filesystem storage?